src/parser/lexer.rs

```diff
@@ -342,7 +342,7 @@ pub const KEYWORDS: &[&str] = &[
 #[derive(Debug, Clone, PartialEq)]
 pub struct SymbolicStringStart<'input> {
     pub prefix: &'input str,
-    pub percent_count: usize,
+    pub length: usize,
```

Comment:
I think we had bug earlier because it's not clear what this quantity really is. Might be useful to either document it, or to name it a bit more explicitly. It seems it's the length of the opening delimiter in total, minux the prefix?

Reply:
That's a good point and yes, that's what it is. I struggle to come up with a better name than `length`, since `percent_count` is definitely wrong. A doc comment is probably the best here.

Reply:
It's well documented now. I think it's ok. Indeed `percent_count` is worst, being deceptive